src/util/currency.js

```diff
@@ -181,14 +181,18 @@ export function formatCurrency(amount, currency, locale) {
     get(currency)
   )(CURRENCY_SYMBOLS);
   const currencyFormat = getCurrencyFormat(currency, locale);
-  const formattedAmount = toCurrencyNumberFormat(amount, currencyFormat);
+
+  const absAmount = amount < 0 ? amount * -1 : amount;
```

Comment:
Using the native [`Math.abs()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs) method would make this a bit more readable: 

```suggestion
  const absAmount = Math.abs(amount);
```